dynoid/middleware/dynoid_test.go

```diff
@@ -0,0 +1,76 @@
+package middleware_test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/go-chi/chi"
+
+	"github.com/heroku/x/dynoid"
+	"github.com/heroku/x/dynoid/dynoidtest"
+	"github.com/heroku/x/dynoid/middleware"
+)
+
+func TestAuthorize(t *testing.T) {
+	router := chi.NewRouter()
+	router.Use(middleware.Authorize(
+		dynoidtest.Audience,
+		dynoid.AllowHerokuHost(dynoidtest.IssuerHost),
+	))
+	router.Get("/", func(w http.ResponseWriter, r *http.Request) {
+	})
```

Comment:
This is a nitpick but we don't need the chi dependency. Apologies if there's a syntax error...

```suggestion
	var handler http.Handler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {})
	handler = middleware.Authorize(
		dynoidtest.Audience,
		dynoid.AllowHerokuHost(dynoidtest.IssuerHost),
	)(handler)
```